src/module/loader/module-resolver.cpp

```diff
@@ -153,7 +160,11 @@ ModuleInfo ModuleResolver::ModuleResolverImpl::Resolve(const char* name, const c
     // Normalize module context path.
     filesystem::Path basePath =
         (path == nullptr) ? filesystem::CurrentDirectory() : filesystem::Path(path);
-    
+
+    if (fromContent) {
+        return ResolveFromContent(name, basePath);
```

Comment:
>ResolveFromContent(name, basePath); [](start = 15, length = 35)

This doesn't make much sense to me. Why would you need to resolve a module (provide it's path) from content where you already know the exact path of the module. In my understanding you shouldn't really call resolve sense it's always the path and a JS module type.